plasma_framework/test/helpers/utils.js

```diff
@@ -31,10 +31,17 @@ function computeNormalOutputId(txBytes, outputIndex) {
     );
 }
 
+function sleep(ms) {
```

Comment:
I would probably use `advanceTime` instead of `sleep` for testing.
https://gist.github.com/ejwessel/76767933280be5cbc59816dae6742661#file-utils-js-L1

Reply:
Even better, OpenZepplin test-helpers `time.increase()` 😄 
https://github.com/OpenZeppelin/openzeppelin-test-helpers